### DISCUSSION TITLE
First Board Meeting of the NewPipe e.V.

### DISCUSSION THREAD

**Discussion by theScrabi** [TeamNewPipe/NewPipe#10212] [Announcements]
Dear NewPipe people and other interested beings,

As you might have heard, last year we founded an association (e.V.), registered in Germany. We gave the board the rule to have a meeting every half year turn. Now, it's time for the first one of these meetings, after our initial founding meeting.
The goal of our e.V. is to support NewPipe and other, similar projects. How we intend this to happen will be decided by the board and the members of the e.V., and then stated on the website of the association. Some examples we can currently come up with are:
- Support the rewrite
- Call in and support live conferences
- Host project infrastructure

Since our founding, we managed to be officially registered by the state of Germany and acquired a bank account there (which took us a while). The next steps will be to setup infrastructure, redirect the money streams to the e.V. and setup a proper way to become a member. Look forward to this happening soon!

Now, if you want to participate in discussions please attend our first online meeting at **13th, 16:00 German time at https://meet.assassinate-you.net/NewPipeEVFirstMeeting**.

We will publish the protocol in case you can not attend to this meeting or do not want to, and will link to it in the comments once we have done so. So if you do not want to miss this, feel free to subscribe to this issue.

Regards, Schabi

**Comment by ghost** [comment]
Super happy about this, bright future ahead 🙌

**Comment by Poolitzer** [comment]
The protocol can be found [here](https://cloud.newpipe-ev.de/s/KxSxfdbbdvfaJav)
